### ISSUE TITLE
Please add support for fcntl standard library

### ISSUE THREAD

**Issue by nirgal** [timothycrosley/isort#265]
If I have these imports:
```
import fcntl
import os
import sys
```

isort change them into:
```
import os
import sys

import fcntl
```

It looks like fnctl is not recognised. Doesn't isort support python3?

Maybe we need a command line argument to choose between the python2 and the python3 set of libraries?

**Comment by urbaniak** [comment]
I've found the same issue.

Looks like resolved here https://github.com/timothycrosley/isort/commit/8895d128b4452196af89902744a7662ff3c1f9e2

Just that needs to be released to pypi

**Comment by timothycrosley** [comment]
Thanks for reporting this issue! This has been fixed in the 4.0 release of isort that has just been pushed to PYPI
